src/components/presentational/CalendarDay/CalendarDay.stories.tsx

```diff
@@ -208,7 +208,19 @@ export const InCalendarNoData = {
                     <CalendarDayInCalendar/>
                 </DateRangeCoordinator>
             </Card>
-        </Layout>
+        </Layout>;
+    }
+};
+
+export const InCalendarNoDataWithClickHandler = {
+    render: () => {
```

Comment:
Consider noting in the onClick parameter description that it is optional, or create a story to handle the DataNoClick handler option.

Reply:
Could you please elaborate a bit?  I'm not sure exactly what you are asking for here.

There are 4 related stories here:
`InCalendarNoData`
`InCalendarNoDataWithClickHandler`
`InCalendarWithData`
`InCalendarWithDataWithClickHandler`

So there are already the two stories that don't have a click handler (indicating its optionality).

Reply:
I'm going to go ahead and merge this so I can pull it into another branch I'm working on.  If you still want an adjustment to the stories, please let me know and I will put them in a follow up PR.

Reply:
My mistake on overlooking these. All good